tests/bluetooth/tester/nrf5340_hci_rpmsg.conf

```diff
@@ -0,0 +1,13 @@
+# Those have to be the same as in the controller (hci_rpmsg)
```

Comment:
Any way we can automatically keep these in sync? 

Also, the `hci_rpmsg` has quite a few different config files for the nRF5340; which one of them are we syncing this file with?

Reply:
Maybe, somehow, do you have any idea?

The hci_rpmsg is built with nrf5340_cpunet_iso-bt_ll_sw_split.conf + CONFIG_BT_CTLR_CONN_ISO_LOW_LATENCY_POLICY=y

Reply:
One way we could do it, is to use the `nrf5340_cpunet_iso-bt_ll_sw_split.conf` as the main config file, and then add whatever we need on top of that as an overlay? 

I don't have any really good suggestions though.